tests/test_subset.py

```diff
@@ -1283,6 +1281,77 @@ def test_get_indexers_nd(self):
             assert ((time_var_name not in indexers.keys()) == True) #time can't be in the index
             assert (new_dataset.dims == dataset.dims)
 
+    def test_variable_type_string_oco2(self):
+        """Code must match the dimensions for each variable rather than assume all dimensions in a group are the same"""
```

Comment:
It looks like this comment doesn't match what this is actually testing -- can you update that?